<launch>
    <arg name="platform" default="sim" />
    <arg name="bond_id_namespace" default="safety_bonds" />

    <node name="motion_planner" pkg="iarc7_planner"
        type="motion_planner"
        launch-prefix="valgrind" >

        <rosparam command="load"
            file="$(find iarc7_planner)/param/platform_$(arg platform).yaml" />
        <rosparam command="load"
            file="$(find iarc7_planner)/param/planner.yaml" />
    </node>

</launch>
